### PR TITLE
Improve general docs

### PR DESCRIPTION
**Pull request by rtorrero**
Update and review main readme.md

I've:
 - updated old information
 - added more detailed instructions
 - removed the examples from the main readme and separated them into directories

### CODE REVIEW

**Review by abravosuse — APPROVED** [review]
After installing the required ansible modules, I am able to kick the execution of the ansible playbook.

File: README.md [hunk 1]
```diff
@@ -16,97 +20,59 @@ This playbook aims to install Trento components and the belonging third parties.
...
-- Obtain the playbook, like git cloning this repository
-- Create an [ansible inventory file](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html)
-- Create a json file containing the variables for the playbook or pass them via cli
-- Run provision command
+Get to the `ansible` directory:
+`cd ansible`
 
-**Example inventory to install the trento-server and provision postgres, rabbitmq and prometheus all on the same host**
+Create an `inventory.yml` file, defining the IP address of the machine where each role will be deployed to. You might use the same machine for more
+than one role. Use `;` to comment out any role that you might not want to cover. Here we assume that we have a valid SSH key in `~/.ssh/id_rsa` to
```
Comment (EMaksy) [inline README.md:53]: What does it mean we assume ? 
```Here we assume that we have a valid SSH key in ~/.ssh/id_rsa to access each of the machines.```

Assuming, sounds a bit soft enough for a technical documentation ?

Maybe putting a requirement or must? 
What do you think?

File: examples/single-node/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+This example portraits a single-node configuration, where all nodes are deployed on the same machine. In this deployment:
```
Comment (EMaksy) [inline examples/single-node/README.md:1]: This example ```portrays``` and not ```portraits``` or ?

File: README.md [hunk 3]
```diff
@@ -118,93 +84,50 @@ all:
...
-```json
-{
-    "trento_server_url:": "http://localhost",
-    "rabbitmq_host": "localhost",
-    "trento_api_key": "api-key-obtained-from-server"
-}
+ansible-playbook -i path/to/inventory.yml --extra-vars "@path/to/vars.json" playbook.yml
 ```
 
 > **Note**: <br />
```
Comment (EMaksy) [inline README.md:133]: Starting with a small letter after Note:

```to have a fully functional deployment make sure to use either an external IP or an internal IP for rabbitmq_host based on the infra network configuration``` 

Maybe we could go with
Note: 
```For a fully functional deployment .....```

**Review by EMaksy — CHANGES_REQUESTED** [review]
Hey man, good job overall.
It is much better to read now!

I left some comments about some tense usage, typos and suggestions.

File: README.md [hunk 4]
```diff
@@ -118,93 +84,50 @@ all:
...
     prometheus-hosts:
       hosts:
-        vitellone-metrics:
+        vitellone:
           ansible_host: "your-host"
           ansible_user: "your-user"
 ```
 
-**Example json variables files to install trento-server, provision postgres, prometheus and rabbitmq, each component on dedicated node**
+### 3. Setup plabook variables:
```
Comment (arbulu89) [inline README.md:102]: Typo `plabook` -> `playbook`

File: README.md [hunk 5]
```diff
@@ -16,97 +20,59 @@ This playbook aims to install Trento components and the belonging third parties.
...
-- SUSE Package Hub 15 SP5 x86_64 - ` SUSEConnect -p PackageHub/15.5/x86_64`
-- Containers Module 15 SP5 x86_64 - `SUSEConnect -p sle-module-containers/15.5/x86_64`
+- Basesystem Module 15 x86_64 - `SUSEConnect -p sle-module-basesystem/15.x/x86_64`
+- SUSE Package Hub 15 x86_64 - ` SUSEConnect -p PackageHub/15.x/x86_64`
+- (Optional: for `docker` installation method) Containers Module 15 x86_64 - `SUSEConnect -p sle-module-containers/15.x/x86_64`
 
 ## Usage
 
-You can clone this repository and use them as a normal ansible playbook, making all the updates you need to make to injected variables, embedding other tasks and so on.
+### 1. Clone the repository:
```
Comment (arbulu89) [inline README.md:43]: I'm always against of putting `:` in headers hehe
But hey, so be it

**Review by arbulu89 — APPROVED** [review]
@rtorrero 
It looks good from my side. Just a couple of small things.
I'm fine when you fix @EMaksy suggestions

File: README.md [hunk 6]
```diff
@@ -16,97 +20,59 @@ This playbook aims to install Trento components and the belonging third parties.
...
+decide which to use.
 
 The `agent` is installed from the configured obs repository using `zypper`.
 
 The nginx configuration acts as a reverse proxy for all the components.
 
 ### SUSE LINUX ENTERPRISE USERS
 
-This playbook assumes you have an activated license of `Suse Linux Enterprise 15 SP5`, with these modules
+This playbook assumes that the server where you are going to install trento-server has an activated license
```
[the remark this answers is not in the document]
Comment (rtorrero) [reply README.md]: I've adjusted the way it's written by making it a requirement. If it's a requirement I think it's implicit that if you don't meet the requirements, the playbook won't work.

File: examples/single-node/README.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+This example portrays a single-node configuration, where all nodes are deployed on the same machine. In this deployment:
+ - `vitellone` is the hostname where `trento-server` gets deployed
+
+Agent nodes are *not* deployed with this configuration.
+
+Two variants are available:
+ - vars-with-alerting.json: In this example, we set up alerting and set the SMTP configuration. With this, important alerts should arrive at the
```
Comment (EMaksy) [inline examples/single-node/README.md:7]: Just a suggestion: 

~~With this, important~~, alerts should arrive at the --> Alerts should arrive at ....

It sounds good, but in the end these are empty words. Also as a reader I start to ask my self, what about the unimportant ? :D

**Review by EMaksy — APPROVED** [review]
Hey man good job, I just left a small nitpick. 
besides this lgtm!

**Pull request merged**